aplib.net-demo/Assets/Scripts/WFC/Cell.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Assets.Scripts.Tiles;
+using System.Collections.Generic;
+
+namespace Assets.Scripts.WFC
+{
+    /// <summary>
+    /// Represents a cell in the grid.
+    /// </summary>
+    public class Cell
+    {
+        /// <summary>
+        /// The tile of the cell.
+        /// </summary>
+        public Tile Tile { get; set; }
+
+        /// <summary>
+        /// The possible tiles that can be placed in this cell.
+        /// </summary>
+        public List<Tile> Candidates { get; set; }
+        
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Cell"/> class.
+        /// </summary>
+        public Cell()
+        {
+            Tile = new Empty();
+            Candidates = new List<Tile>()
+            {
+                new Corner(),
+                new Corner(1),
+                new Corner(2),
+                new Corner(3),
+                new Crossing(),
+                new DeadEnd(),
+                new DeadEnd(1),
+                new DeadEnd(2),
+                new DeadEnd(3),
+                new Empty(),
+                new Straight(),
+                new Straight(1),
+                new TSection(),
+                new TSection(1),
+                new TSection(2),
+                new TSection(3),
+            };
```

Comment:
I understand you have to test this somehow, but can this be created somewhere else? It could be a different class that stores these things, it could be serialised, or it could be set from the Unity editor, as that supports lists to be added through the editor.

Reply:
Adding onto this: if it is deleted upon finalising the WFC, this can stay. (And if that is the case, please do respond here but you can resolve the conversation)

Reply:
It is deleted when the WFC is finalised.